traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/url_sig_keys.go

```diff
@@ -30,26 +30,31 @@ import (
 	"github.com/jmoiron/sqlx"
 )
 
-func getURLSigKeys(xmlID string, tvTx *sqlx.Tx, ctx context.Context) (tc.URLSigKeys, bool, error) {
-	var jsonUrlKeys string
-	if err := tvTx.QueryRow("SELECT data FROM url_sig_key WHERE deliveryservice = $1", xmlID).Scan(&jsonUrlKeys); err != nil {
+func getURLSigKeys(xmlID string, tvTx *sqlx.Tx, ctx context.Context, encryptionKeyLocation string) (tc.URLSigKeys, bool, error) {
+	var encryptedUrlSigKey string
```

Comment:
Would it make sense to have this be `[]byte` instead of a `string`? Since the encryption/decryption works w/ `[]byte` and the DB type is `bytea`?